include/mbedtls/config.h

```diff
@@ -2608,7 +2608,7 @@
  *
  * \warning This is currently experimental. EC J-PAKE support is based on the
  * Thread v1.0.0 specification; incompatible changes to the specification
- * might still happen. For this reason, this is disabled by default.
```

Comment:
This was written in 2015. Is this option still experimental?
If so, why enable it by default?
If not, this warning should be removed.

Reply:
Done

Reply:
Good point. I think this should be replaced with a different text. Suggestion:
```
\note EC J-PAKE support is based on the Thread v1.0.0 specification. It has not been reviewed for compliance with newer standards such as Thread v1.1 or RFC 8236.
```
I've also created https://github.com/ARMmbed/mbedtls/issues/4467 to track that.

Reply:
Done